test/output.js

```diff
@@ -0,0 +1,76 @@
+import test from 'ava';
```

Comment:
Shouldn't this be `import {serial as test} from 'ava';` to make sure it runs all the tests serially. Otherwise the output could be mangled. It's weird that it works right now to be honest.